src/AddTodoForm.js

```diff
@@ -22,15 +23,12 @@ export default function AddTodoForm (props) {
     return(
         <>
             <form onSubmit={handleAddTodo}>
-                <label className="side--marg bold" htmlFor="todoTitle">Title</label>
-                <input 
-                    className="side--marg" 
-                    id="todoTitle" 
-                    name="title"
-                    value = {todoTitle}
-                    onChange = {handleTitleChange}
-                >
-                </input>
+                <InputWithLabel
+                    todoTitle = {todoTitle}
+                    handleTitleChange = {handleTitleChange}
+                    label = "Title"
+                    isFocused
```

Comment:
I see label and isFocused passed into InputWithLabel. I do not think you need label because it is static. Also, isFocused will always be true in this situation.